ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -424,6 +449,28 @@ static void guarded_move_done_cb
   publish ("GroundPosition", GroundPosition);
 }
 
+//Since template is outside the scope of the class we cannot use a member variable and 
+//instead have to use a global one like in the guarded_move_done_cb above.
+static std::vector<double> m_sample_point;
+static bool got_sample_location_result = false;
```

Comment:
I'm not sure if we have a naming standard for static variables, but "m_" should be reserved for classes.  I'll suggest SamplePoint and GotSampleLocationResult to be consistent with the other static variables in this file/package.